internal/proxy/task_scheduler.go

```diff
@@ -650,10 +651,13 @@ func (sched *taskScheduler) collectResultLoop() {
 	queryResultMsgStream.Start()
 	defer queryResultMsgStream.Close()
 
+	const bufFlagExpireTime = 5 * time.Minute
```

Comment:
please change the constants to a param. bufFlagExpireTime could be a little bit longer such as 1 hour~

Reply:
done